change/@ni-nimble-components-8ca8762f-ad08-436d-980f-56d9f5ba9e39.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
While using my mousewheel I get into a state (inconsistently) where the hover stops working:

![hovereffect](https://github.com/user-attachments/assets/93fed7d9-4950-46af-9743-2d3e5fd8c3a8)

Reply:
Weird. I had seen something like that myself, but it was only when I got to the very top or bottom. This is clearly more pervasive.

Reply:
Ugh. Looks like there are a couple browser bugs involved 😭
![scrollend-bug-chrome](https://github.com/user-attachments/assets/fd3a47a9-5fa0-4959-a651-cfbd66e77172)
Above is Chrome. In some cases the `scrollend` event doesn't get fired. It seems like mouse movement is a necessary component of the bug.

On Firefox, we very frequently get a `scroll` event (indicated by a `.`) _after_ the `scrollend` event, which effectively looks like an open-ended scroll to TanStack.
![scrollend-bug-firefox](https://github.com/user-attachments/assets/2fc77c2e-0afa-4e4c-9b2e-c83805b436a0)
https://codepen.io/m-akinc/pen/gONQawN

I haven't filed browser bugs, but I will.

In this case, I think a reasonable workaround is to simply sett `isScrolling` to false if we haven't gotten a virtualizer change notification in 300ms (we get one for each `scroll` event). Let me know if you still see any weirdness.

Reply:
I added my report to an [existing Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1896515#c4).
Filed [this bug](https://issues.chromium.org/issues/364568425) to Chromium.